voce-post-meta.php

```diff
@@ -418,10 +419,15 @@ public function update_field( $post_id ) {
 			if ( is_callable( $callback ) )
 				$new_value = call_user_func( $callback, $this, $old_value, $new_value, $post_id );
 		}
+
 		if ( is_null($new_value) || false === $new_value || "" === $new_value ) {
-			delete_post_meta( $post_id, "{$this->group->id}_{$this->id}" );
+			delete_post_meta( $post_id, $meta_key );
+
+			do_action( 'voce_meta_deleted_field', $post_id, $meta_key, $new_value, $old_value, $this );
```

Comment:
Figured that it might make a difference to someone whether the value is null vs false vs "" (perhaps to be able to compare what the validation returns with what was submitted in the `$_POST`), and to keep the arguments consistent with the `updated` filter. Keep/remove?